atomic_reactor/core.py

```diff
@@ -95,6 +96,8 @@ def _obtain_source_from_path_if_needed(self, local_path, container_path=CONTAINE
         build_json_path = os.path.join(local_path, BUILD_JSON)
         with open(build_json_path, 'r') as fp:
             build_json = json.load(fp)
+
+        validate(build_json, JSON_SCHEMA)
```

Comment:
Did you come to a conclusion about whether this could be made to give a better pointer to where the problem is?